src/Tizen.Multimedia.MediaPlayer/Player/MediaStreamSource.cs

```diff
@@ -42,7 +44,39 @@ public static IEnumerable<MediaFormatAudioMimeType> SupportedAudioTypes
         {
             get
             {
-                yield return MediaFormatAudioMimeType.Aac;
+                NativePlayer.Create(out _playerHandle).ThrowIfFailed(null, "Failed to create player");
+                _handle = _playerHandle.DangerousGetHandle();
+                Debug.Assert(_handle != IntPtr.Zero);
+
+                try
+                {
+                    List<MediaFormatAudioMimeType> audioFormats = new List<MediaFormatAudioMimeType>();
```

Comment:
How about the following?
```csharp
private static List<MediaFormatAudioMimeType> _supportedAudioFormats;
public static IEnumerable<MediaFormatAudioMimeType> SupportedAudioTypes
{
     get
      {
            if (_supportedAudioFormats== null)
            {
                  _supportedAudioFormats= new List<MediaFormatAudioMimeType>();
                  ...
            }
            
            return _videoFormats;
      }
```

Reply:
ok i'll keep it into local variable

Reply:
`_supportedAudioFormats` should be captured by `field`, not local variable.

Reply:
right i meant that. i fixed it to a field variable.